x-pack/plugins/cases/common/ui/types.ts

```diff
@@ -126,6 +126,7 @@ export interface ElasticUser {
   readonly email?: string | null;
   readonly fullName?: string | null;
   readonly username?: string | null;
+  readonly profileUid?: string | null;
```

Comment:
nit: Is it possible to use `SnakeToCamelCase<User>` to define the `ElasticUser` interface?